kubectl-cloudflow/cmd/deploy.go

```diff
@@ -53,25 +53,25 @@ is the name of a Kubernetes Persistent Volume Claim, which needs to be located
 in the same namespace as the Cloudflow application, e.g. the namespace with the
 same name as the application.
 
-  kubectl cloudflow deploy docker.registry.com/my-company/sensor-data-scala:292-c183d80 --volume-mount my-streamlet.mount=pvc-name
+  kubectl cloudflow deploy call-record-aggregator-cr.json --volume-mount my-streamlet.mount=pvc-name
 
 It is also possible to specify more than one "volume-mount" parameter.
 
-  kubectl cloudflow deploy docker.registry.com/my-company/sensor-data-scala:292-c183d80 --volume-mount my-streamlet.mount=pvc-name --volume-mount my-other-streamlet.mount=pvc-name
+  kubectl cloudflow deploy call-record-aggregator-cr.json --volume-mount my-streamlet.mount=pvc-name --volume-mount my-other-streamlet.mount=pvc-name
 
 You can optionally provide credentials for the docker registry that hosts the
-specified image by using the --username flag in combination with either
+images of the application by using the --username flag in combination with either
 the --password-stdin or the --password flag.
 
 The --password-stdin flag is preferred because it is read from stdin, which
 means that the password does not end up in the history of your shell.
 One way to provide the password via stdin is to pipe it from a file:
 
-  cat key.json | kubectl cloudflow deploy docker.registry.com/my-company/sensor-data-scala:292-c183d80 --username _json_key --password-stdin
+  cat key.json | kubectl cloudflow deploy call-record-aggregator-cr.json --username _json_key --password-stdin
 
 You can also use --password, which is less secure:
 
-  kubectl cloudflow deploy docker.registry.com/my-company/sensor-data-scala:292-c183d80 --username _json_key -password "$(cat key.json)"
+  kubectl cloudflow deploy call-record-aggregator-cr.json --username _json_key -password "$(cat key.json)"
```

Comment:
minor: not sure if we add the '-cr', but we could drop that.

Reply:
done ..